client/package.json

```diff
@@ -15,7 +15,7 @@
     "rollup-plugin-node-resolve": "^5.0.0",
     "rollup-plugin-terser": "^4.0.4",
     "tslint": "^5.16.0",
-    "typescript": "^3.4.5"
+    "typescript": "^3.5.1"
```

Comment:
Didn't do so to avoid an incompatibility between new `typescript` version and `@types/node` which is a sub dependency of other dependencies.
I now updated both of them.